### ISSUE TITLE
No badge

### ISSUE THREAD

**Issue by donag0522** [keybase/client#21144]
Hi, i registered in keybase sometime in September but till now i dont have a badge?.. I want to join airdrop

**Comment by munnamizan1494** [comment]
> Hi, i registered in keybase sometime in September but till now i dont have a badge?.. I want to join airdrop

Same problem bro :( I have a badge when I register on sept. Now no badge and no airdrop. :((

**Comment by donag0522** [comment]
Yeah...it is so sad so see them enjoying the airdrop while we are also a keybase member but we dont have badge and cant join airdrop
